packages/11ty/_includes/components/lightbox/index.js

```diff
@@ -0,0 +1,28 @@
+const { html } = require('common-tags')
+
+const stringifyData = (jsObject) => {
+  return encodeURIComponent(JSON.stringify(jsObject));
+}
```

Comment:
**TODO**: replace this object string-ification function with [yahoo-serialize library](https://github.com/yahoo/serialize-javascript)